xmake/toolchains/xcode/load_macosx.lua

```diff
@@ -19,69 +19,71 @@
 --
 
 function main(toolchain)
+    -- init architecture
+    local arch = toolchain:arch()
+    local xcode_sdkver = toolchain:config('xcode_sdkver') or get_config("xcode_sdkver")
```

Comment:
we need not `or get_config("xcode_sdkver")`, xcode/check will get and save it to toolchain:config